annotation-utils/src/main/java/edu/isi/vista/annotationutils/ExportAnnotations.kt

```diff
@@ -123,12 +123,9 @@ class ExportAnnotations {
                 val projectUsername = if (userSeparatorIndex >= 0)
                     project.name.substring(userSeparatorIndex + 1, project.name.length) else null
                 var finalProjectName = project.name
-                var outputUsername = projectUsername
-                if (usernameMap != null) {
-                    outputUsername = usernameMap.get(projectUsername)?.toString()?.removeSurrounding("\"")
-                    if (projectUsername != null && outputUsername != null) {
-                        finalProjectName = project.name.substring(0, userSeparatorIndex + 1) + outputUsername
-                    }
+                val outputUsername = usernameMap.get(projectUsername)?.toString()?.removeSurrounding("\"")
+                if (projectUsername != null && outputUsername != null) {
+                    finalProjectName = project.name.substring(0, userSeparatorIndex + 1) + outputUsername
```

Comment:
Is there anywhere in here where the format of the files is explicitly said in a comment? Would be informative for this code

Reply:
I could certainly add it